lib/index.js

```diff
@@ -36,13 +38,22 @@ var method = module.exports = (obj, key, method) => {
   }
   flag.add(key);
 
-  Object.defineProperty(obj, key, {
-    writable: true,
-    configurable: true,
-    enumerable: true,
-    value: method
-  });
-
+  if (!method && extend) {
+    // method->null use extend
+    Object.defineProperty(obj, key, {
+      configurable: true,
+      enumerable: true,
+      get: extend.getter || noop,
+      set: extend.setter || noop,
```

Comment:
What do you think of naming these `get` and `set` so that it matches more closely to the object being extended?